server/controllers/items/inventory_view.js

```diff
@@ -27,15 +27,14 @@ const validateUserOrGroup = params => {
 }
 
 const getItems = async params => {
-  const { user, group, shelf, reqUserId, 'without-shelf': withoutShelf } = params
-  if (user) {
-    return getAuthorizedItems.byUser(user, reqUserId, { withoutShelf })
-  }
-  if (shelf) {
-    const shelfDoc = await shelves_.byId(shelf)
-    return getAuthorizedItems.byShelf(shelfDoc, reqUserId)
+  const { userId, groupId, shelfId, reqUserId, 'without-shelf': withoutShelf } = params
```

Comment:
Sanitization implicitly adding the suffix `Id` to `user` `group` and `shelf` is a bit confusing to me. "Where does that `Id` comes from ?" stopped me in the review, and i guess a newcomer could also block on this.

I would rely on JS convience to make it an explicit declaration: 

```suggestion
  const { user: userId, group: groupId, shelf: shelfId, reqUserId, 'without-shelf': withoutShelf } = params
```

Reply:
done in a8967657